lib/opte/src/ddi/time.rs

```diff
@@ -75,6 +77,19 @@ impl Moment {
             }
         }
     }
+
+    /// Return the underlying timestamp for debugging purposes
+    /// if supported on the current platform.
+    #[allow(dead_code)]
```

Comment:
I presume this is because we're only calling this in a `no-std` context?

Reply:
It is, yes.